include/dlaf/communication/kernels/broadcast.h

```diff
@@ -29,55 +29,28 @@
 #include <dlaf/types.h>
 
 namespace dlaf::comm {
-namespace internal {
-template <class T, Device D>
-void sendBcast(const Communicator& comm, const matrix::Tile<const T, D>& tile, MPI_Request* req) {
-#if !defined(DLAF_WITH_MPI_GPU_AWARE)
-  static_assert(D == Device::CPU, "DLAF_WITH_MPI_GPU_AWARE=off, MPI accepts only CPU memory.");
-#endif
-
-  auto msg = comm::make_message(common::make_data(tile));
-  DLAF_MPI_CHECK_ERROR(MPI_Ibcast(const_cast<T*>(msg.data()), msg.count(), msg.mpi_type(), comm.rank(),
-                                  comm, req));
-}
-
-DLAF_MAKE_CALLABLE_OBJECT(sendBcast);
-
-template <class T, Device D>
-void recvBcast(const Communicator& comm, comm::IndexT_MPI root_rank, const matrix::Tile<T, D>& tile,
-               MPI_Request* req) {
-#if !defined(DLAF_WITH_MPI_GPU_AWARE)
-  static_assert(D == Device::CPU, "DLAF_WITH_MPI_GPU_AWARE=off, MPI accepts only CPU memory.");
-#endif
-
-  auto msg = comm::make_message(common::make_data(tile));
-  DLAF_MPI_CHECK_ERROR(MPI_Ibcast(msg.data(), msg.count(), msg.mpi_type(), root_rank, comm, req));
-}
-
-DLAF_MAKE_CALLABLE_OBJECT(recvBcast);
-}
 
 /// Schedule a broadcast send.
 ///
 /// The returned sender signals completion when the send is done. If the input
 /// tile is movable it will be sent by the returned sender. Otherwise a void
 /// sender is returned.
 template <class T, Device D, class Comm>
-[[nodiscard]] pika::execution::experimental::unique_any_sender<> scheduleSendBcast(
+[[nodiscard]] pika::execution::experimental::unique_any_sender<> schedule_bcast_send(
```

Comment:
No objection to the name change, I just wanted to ask what the motivation for the change was? (I'm referring to `bcast send` vs `send bcast`, not the change of case).

Reply:
To have a consistent naming among the different collectives.
`scheduleSendBcast` vs `scheduleReduceSend`
I chose the second one, but can go with the first option as well if you think is better.

Reply:
That makes sense, thanks!